src/test/kotlin/com/coxautodev/graphql/tools/EndToEndSpecHelper.kt

```diff
@@ -31,6 +31,7 @@ val schemaDefinition = """
 ## Private comment!
 scalar UUID
 scalar customScalarMap
+scalar Upload
```

Comment:
I believe some testing code was left behind in this PR.

Reply:
The line is related to issue. The scalar is used in 'echoFiles' mutation.

Reply:
Let me know if there is anything need.